docs/validation/validators.md

```diff
@@ -135,7 +135,46 @@ const time = {
   second: null // Valud missing, but this will be ignored.
 }
 
-hms(time, ['h', 'm']) // <- true
+requiredHms(time, ['h', 'm']) // <- true
+```
+
+## `requiredHmsIf`
+
+Checks if the given hour, minute and second is present when the condition passed is truthy. This validator will not validate if the values are valid numbers. To check the validity of the value, use [`hms`](#hms) validator.
+
+```ts
+import { type Hms } from '@globalbrain/sefirot/lib/support/Day'
+import { type RequiredIfCondition } from '@globalbrain/sefirot/lib/validation/validators'
+
+function requiredHms(
+  hms: Hms,
+  condition: RequiredIfCondition
+  required: ('h' | 'm' | 's')[] = ['h', 'm', 's']
+): boolean
+```
+
+```ts
+const time = {
+  hour: '10',
+  minute: null, // Value missing.
+  second: '00'
+}
+
+requiredHmsIf(time, true) // <- false
+requiredHmsIf(time, false) // <- true
+```
+
+You may pass 3rd argument to specify which fields to validate. For example,if you pass `['h', 'm']`, it will validate only hour and minute.
```

Comment:
```suggestion
You may pass 3rd argument to specify which fields to validate. For example, if you pass `['h', 'm']`, it will validate only hour and minute.
```

Reply:
Fixed: 46e2656